src/test/shell/bazel/external_patching_test.sh

```diff
@@ -42,6 +64,12 @@ EOF
 test_patch_file() {
   EXTREPODIR=`pwd`
 
+  if is_windows; then
```

Comment:
Could you create a helper function to compute EXTREPOURL for a given path?

Reply:
Done.